assets/src/components/cd/cluster/ClusterMetadata.tsx

```diff
@@ -80,7 +86,23 @@ function MetadataCard({
               }
             />
           </Prop>
-
+          {!cluster?.self && (
+            <Prop
+              title="Service"
+              margin={0}
+            >
+              <A
+                as={Link}
+                to={getServiceDetailsPath({
+                  clusterId: cluster?.id,
+                  serviceId: cluster?.service?.id,
+                })}
+                inline
+              >
+                {cluster?.service?.name}
+              </A>
```

Comment:
Should avoid adding new instances of honorable components. There's `components/utils/typography/InlineLink.tsx`  that could be used.

Reply:
You're right. That was quick copy paste, I will update it.

Reply:
Done, thanks for pointing it out.